From 686fae67829e4d8b9d768d88636f25fcdc3a2d9f Mon Sep 17 00:00:00 2001
From: Sugar Zhang <sugar.zhang@rock-chips.com>
Date: Tue, 1 Sep 2020 17:52:24 +0800
Subject: [PATCH] Revert "ARM: dts: rockchip: rv1126 io-domain node add VCCIO1"

This reverts commit fae0c499d0af038442b4030ddbca1378a1dadaec.

VCCIO1 is controlled by pin GPIO0B3/FLASH_VOL_SEL:

* pullup for 1.8v
* float for 3.3v

so, we should not reconfigure it by software.

Signed-off-by: Sugar Zhang <sugar.zhang@rock-chips.com>
Change-Id: I0bc119cbb0355e9d8fd7ba3457bf436eba3d6e7a
---
 arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts   | 1 -
 arch/arm/boot/dts/rv1126-38x38-v10-spi-nor.dts | 1 -
 arch/arm/boot/dts/rv1126-ai-cam.dtsi           | 1 -
 arch/arm/boot/dts/rv1126-bat-ipc-v10.dts       | 1 -
 arch/arm/boot/dts/rv1126-evb-v10.dtsi          | 1 -
 arch/arm/boot/dts/rv1126-iotest-v10.dts        | 1 -
 6 files changed, 6 deletions(-)

diff --git a/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts b/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
index 4b964970c2d2..6307fc64d197 100644
--- a/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
+++ b/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
@@ -195,7 +195,6 @@
 
 	pmuio0-supply = <&vcc3v3_sys>;
 	pmuio1-supply = <&vcc3v3_sys>;
-	vccio1-supply = <&vcc_1v8>;
 	vccio2-supply = <&vcc3v3_sys>;
 	vccio4-supply = <&vcc_1v8>;
 	vccio5-supply = <&vcc3v3_sys>;
diff --git a/arch/arm/boot/dts/rv1126-38x38-v10-spi-nor.dts b/arch/arm/boot/dts/rv1126-38x38-v10-spi-nor.dts
index 7ef48f04acc3..47ee81a2e743 100644
--- a/arch/arm/boot/dts/rv1126-38x38-v10-spi-nor.dts
+++ b/arch/arm/boot/dts/rv1126-38x38-v10-spi-nor.dts
@@ -334,7 +334,6 @@
 
 	pmuio0-supply = <&vcc3v3_sys>;
 	pmuio1-supply = <&vcc3v3_sys>;
-	vccio1-supply = <&vcc_1v8>;
 	vccio2-supply = <&vcc3v3_sys>;
 	vccio4-supply = <&vcc_1v8>;
 	vccio5-supply = <&vcc3v3_sys>;
diff --git a/arch/arm/boot/dts/rv1126-ai-cam.dtsi b/arch/arm/boot/dts/rv1126-ai-cam.dtsi
index 4d3ae9f04fd1..826dc8b12a49 100644
--- a/arch/arm/boot/dts/rv1126-ai-cam.dtsi
+++ b/arch/arm/boot/dts/rv1126-ai-cam.dtsi
@@ -211,7 +211,6 @@
 
 	pmuio0-supply = <&vcc_3v3>;
 	pmuio1-supply = <&vcc_3v3>;
-	vccio1-supply = <&vcc_1v8>;
 	vccio2-supply = <&vcc_3v3>;
 	vccio3-supply = <&vcc_1v8>;
 	vccio4-supply = <&vcc_1v8>;
diff --git a/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts b/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
index 255121300cc5..9372d6ab629a 100644
--- a/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
+++ b/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
@@ -173,7 +173,6 @@
 
 	pmuio0-supply = <&vcc_1v8>;
 	pmuio1-supply = <&vcc_3v3>;
-	vccio1-supply = <&vccio_flash>;
 	vccio2-supply = <&vccio_sd>;
 	vccio3-supply = <&vcc_1v8>;
 	vccio4-supply = <&vcc_1v8>;
diff --git a/arch/arm/boot/dts/rv1126-evb-v10.dtsi b/arch/arm/boot/dts/rv1126-evb-v10.dtsi
index 061a7b15258a..396ef1516054 100644
--- a/arch/arm/boot/dts/rv1126-evb-v10.dtsi
+++ b/arch/arm/boot/dts/rv1126-evb-v10.dtsi
@@ -1096,7 +1096,6 @@
 
 	pmuio0-supply = <&vcc1v8_pmu>;
 	pmuio1-supply = <&vcc3v3_sys>;
-	vccio1-supply = <&vcc_1v8>;
 	vccio2-supply = <&vccio_sd>;
 	vccio3-supply = <&vcc_1v8>;
 	vccio4-supply = <&vcc_1v8>;
diff --git a/arch/arm/boot/dts/rv1126-iotest-v10.dts b/arch/arm/boot/dts/rv1126-iotest-v10.dts
index 197a8f5fa36a..420b8822fbc5 100644
--- a/arch/arm/boot/dts/rv1126-iotest-v10.dts
+++ b/arch/arm/boot/dts/rv1126-iotest-v10.dts
@@ -331,7 +331,6 @@
 
 	pmuio0-supply = <&vcc1v8_pmu>;
 	pmuio1-supply = <&vcc3v3_sys>;
-	vccio1-supply = <&vcc_1v8>;
 	vccio2-supply = <&vcc_3v3>;
 	vccio3-supply = <&vcc_3v3>;
 	vccio4-supply = <&vcc_3v3>;
-- 
2.35.3

